_pytest/cacheprovider.py

```diff
@@ -186,8 +186,20 @@ def pytest_configure(config):
 
 @pytest.fixture
 def cache(request):
+    """
+    Return a cache object that can persist state between testing sessions.
+
+    cache.get(key, default)
+    cache.set(key, value)
+
+    Keys must be strings not containing a "/" separator. Add a unique identifier
```

Comment:
oh darn, the docstring is ctually in reversed logic, i completely missed it - @nicoddemus ?